log_functions.py

```diff
@@ -47,3 +54,28 @@ def log_verbosity_to_logging(verbosity: str):
 def handle_error_and_exit(error: typing.Union[str, Exception]):
     logging.error(str(error))
     sys.exit(1)
+
+
+def send_email(error: str, date_time: str = None) -> None:
+    smtp_conn = smtplib.SMTP(config.notification.smtp_server)
+    smtp_conn.login(config.notification.smtp_username, config.notification.smtp_password)
+
+    msg = MIMEMultipart()
+    msg["Subject"] = config.notification.email_subject
+    msg["From"] = config.notification.email_sender
+    msg["To"] = ", ".join(config.notification.email_recipients)
+    msg.preamble = error
+
+    if date_time:
+        error = f"{date_time}: {error}"
```

Comment:
How much context is there in `error` ? A common issue may be that database schema has been modified. How would such an email look like?

Reply:
The error may look like:
```
26/07/2023 11:59:36: geodiff failed!
['geodiff', 'diff', '--driver', 'postgres', 'host=localhost dbname=mergin_dbsync user=admin password='*****' port=5430', 'db_sync_base', 'db_sync_main', '/tmp/db-sync-dbsync-push-base2our']
```